app/controllers/api/v1/users_controller.rb

```diff
@@ -11,7 +11,6 @@ def index
 
       def create
         user = User.new(user_params)
-
```

Comment:
Small nit.  Please keep this space in between the group of assigning variables, and taking actions.